apps/api/src/app/subscribers-v2/e2e/get-subscriber-preferences.e2e.ts

```diff
@@ -51,61 +46,44 @@ describe('Get Subscriber Preferences - /subscribers/:subscriberId/preferences (G
     expect(response.statusCode).to.equal(404);
   });
 
-  it('should handle subscriber with modified workflow preferences', async () => {
```

Comment:
I removed overlapping tests from GET endpoint and replaced them with tests testing unique GET behaviour